### PR TITLE
create and use default rt across tests

### LINKED ISSUE #287: Re-use simple Rt definition throughout testing suite
see e.g.:

> Should we share this across multiple files?

_Originally posted by @damonbayer in https://github.com/CDCgov/multisignal-epi-inference/pull/275#discussion_r1685019213_

### PR DESCRIPTION
**Pull request by sbidari**
closes #287

### CODE REVIEW

File: model/src/test/utils.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+test utilities
+"""
+
+import numpyro.distributions as dist
+import pyrenew.transformation as t
+from pyrenew.metaclass import DistributionalRV, TransformedRandomVariable
+from pyrenew.process import SimpleRandomWalkProcess
+
+
+def get_default_rt():
```
Comment (damonbayer) [inline model/src/test/utils.py:13]: suggest renaming. Something like "simple_rt()". Also might be useful to have arguments for names (with the same defaults as here). That way you could use the function twice in the same model if you needed to.

**Review by damonbayer — CHANGES_REQUESTED** [review]
This will be really nice. Two suggestions.

File: model/src/test/utils.py (hunk 1 above)
Comment (sbidari) [reply model/src/test/utils.py]: changed in 875e28392516bc6ba06fb2de210368a4dfad68fe

**Review by damonbayer — APPROVED** [review]
Thanks @sbidari!

**Pull request merged**